lib/libutee/tee_api_objects.c

```diff
@@ -434,40 +432,40 @@ TEE_Result TEE_CreatePersistentObject(uint32_t storageID, const void *objectID,
 	TEE_Result res;
 	uint32_t obj;
 
+	if (!object)
+		TEE_Panic(TEE_ERROR_BAD_PARAMETERS);
```

Comment:
Actually, this is not needed. The TA will panic from a segfault if pointer is null.
Suggest to remove. Despite my R-b tag.